packages/utilities/src/index.tsx

```diff
@@ -3,6 +3,7 @@ export { default as arrayRandom } from "./arrayRandom";
 export { default as arrayUnique } from "./arrayUnique";
 export { default as capitalize } from "./capitalize";
 export { default as For, ForProps } from "./For";
+export * as useHooks from "./hooks";
```

Comment:
sa `export * as useHooks` @jade-kenneth, pagnag import tau sa package bali ganito na `import { useHooks } from '<hooks package>'` ? d ata to sia matree-shake?